snykTask/src/__tests__/test-task-args.ts

```diff
@@ -44,6 +44,23 @@ test('ensure no problems if both targetFile and docker-file-path are both not se
   expect(fileArg).toBe('');
 });
 
+test('ensure no problems if testType set to code and both targetFile and dockerImageName are both set', () => {
+  const args = defaultTaskArgs();
+  args.testType = TestType.CODE;
+  args.dockerImageName = 'some-docker-image';
+  args.targetFile = 'some-target-file';
+  args.dockerfilePath = null as any;
```

Comment:
Looking at the `TaskArgs` class, I noticed that `targetFile` and `dockerfilePath` are optional and should either be strings or `undefined`. The use of null as `any` is not great since it bypasses the type safety check. 

Is is possible to remove `as any` and replace with `args.dockerfilePath = undefined;`?

Reply:
corrected